phoenix-core/src/main/java/org/apache/phoenix/schema/PTable.java

```diff
@@ -341,4 +345,13 @@ public static LinkType fromSerializedValue(byte serializedValue) {
      */
     int getRowTimestampColPos();
     long getUpdateCacheFrequency();
+
+    boolean isNamespaceMapped();
+
+    /**
+     * For a view, return the name of table in Phoenix that physically stores data.
+     * 
+     * @return the name of the Phoenix table storing the data.
+     */
+    PName getPhoenixPhysicalName();
```

Comment:
Can you tell me more about this new method? What is its purpose? Looking at the comments it looks like exactly same as getPhysicalName()